### ISSUE TITLE
Home page in a course : problem with 2 & 3 columns view

### ISSUE THREAD

**Issue by tanchon** [chamilo/chamilo-lms#2255]
### Current behavior / Resultado actual / Résultat actuel

Problems in a course with some "views".

**No problem with the "Activity view", "Big Activity view (ipad), "Vertical activity view" :**

- when the admin platform decided to hide some tools in "Hide tools from teachers", these tools are really desactivated : it is impossible for the teacher to reactivate these tools (he cannot see them !) ;

- when the admin platform decided to desactivate some tools in "Modules active upon course creation", these tools are visible for the teachers (in grey) (ONLY if they were not hidden by the admin platform) and it is possible for the teachers to make them visible for the students.

But there is a problem for the "Two columns view" and "Three columns view" :

**"2 columns view"  :**

- when the admin platform decide to hide some tools in "Hide tools from teachers", these tools are NOT REALLY hidden : they are visible (in grey) and the teachers can activate them (and make them visible).  When the teachers activate these tools, they really become active, even for students (if the teachers don't reactivate the tools, the students cannot see them).

- when the admin platform decide to desactivate some tools in "Modules active upon course creation", these tools are visible for the teachers (in grey) if they were not hidden by the admin platform) and it is possible for the teachers to reactivate. It's OK. BUT THERE IS NO DIFFERENCE between these desactivated tools and the "hidden tools"...

**"3 columns view"**  : 
The problem is the same as "2 columns view" : the only difference is for students => they can see the tools that the admin platform decided to hide in "Hide tools from teachers" (but the tools appears in grey and are not active... except if the teachers reactivate them) !

### Expected behavior / Resultado esperado / Résultat attendu

For the "2 columns view"  & "3 columns view" , the behavior should be the same as for the other views (activity view...) : i described it at the beginning of my post.

### Chamilo Version / Versión de Chamilo / Version de Chamilo

**Problem with Chamilo 1.11.2 AND Chamilo 1.11.4**

**Comment by ywarnier** [comment]
Only the "Big activity" view is supported by the Chamilo development team. We are leaving the other options because some very long-term portals are still using them, but using the 2-3 column views is not recommended. I understand that coming from Claroline, you would like them, but we don't have enough resources to be offering support for these for free.

**Comment by ywarnier** [comment]
Other modes should be rewritten with TPLs so that we can reasonnably maintain them.

**Comment by aragonc** [comment]
@ywarnier  esto se refiere a la forma como se presenta las herramientas del curso, verdad?

**Comment by ywarnier** [comment]
@aragonc correcto. La primera opción dentro de la categoría "Cursos" de parámetros de configuración de la plataforma.

**Comment by ywarnier** [comment]
@aragonc falta tpl-izar las distintas vistas
